app/url_image_scraper.py

```diff
@@ -100,8 +100,8 @@ def scrape_URL(case_study_name, job_id, tracer_id, scraped_data_repository, log_
 
                         relative_path = f"{case_study_name}/{tracer_id}/{job_id}/{unix_timestamp}/webcam/{data_name}.{file_extension}"
 
-                        next_capture_time += interval
-                        num_screenshots+=1
+                        current_date += interval
```

Comment:
@Rahul-7131 if you're adding the interval here, why does the computation of the unix timestamp has a `+ interval` ? in line 92